app/common/directives/filter-system/filter-role.html

```diff
@@ -7,8 +7,7 @@
                 checklist-model="model"
                 checklist-value="role.name"
                 >
-            {{role.display_name}}
+            {{role.display_name | translate}}
```

Comment:
Isn't display name user editable? Any reason users can't just edit the role names to match their language?

Reply:
I added this so we can translate the pre-added role-names. If we support many different languages, I think we should translate those as well? Same as with the pre-added saved searches. However, for this specific client, we could do it manually. Leaving this in prepares for translating the role-names, it still doesnt mess up if a user edits it (if they don't edit it to one of our translation-ids, which I think is unlikely)

Reply:
I finally removed these translations since they can be done in the api instead

Reply:
Ok. The experience will be a bit weird when editing though. ie. if we ship a translation for 'Admin' to French, but the user goes to edit the Admin role. They'll see the original English version in the edit view, not the French.
Anyway - a real solution has to wait till we have time to make roles/categories and other user editable translatable in the API - in the interrim this makes it easier to support clients I guess